src/licence-status-banner/index.jsx

```diff
@@ -26,15 +26,23 @@ function LicenceStatusBanner({ licence, licenceType, isPdf, dateFormat, colour,
         }
 
         return <ul className="licence-dates">
-            <li>Granted: <span className="date">{format(issueDate, dateFormat)}</span></li>
             {
-                status === 'revoked' && <li>Revoked: <span className="date">{format(revocationDate, dateFormat)}</span></li>
+                issueDate != null && <li>Granted: <span className="date">{format(issueDate, dateFormat)}</span></li>
```

Comment:
Will there ever be a case of undefined?

Reply:
There are at least some projects in the end-to-end tests with an undefined or null granted date - not sure if inaccurate test data or if there's a scenario where a draft licence can be revoked/suspended/expired.